storage/storage.go

```diff
@@ -1362,7 +1362,7 @@ func (q *Query) SetAttrSelection(attrs []string) error {
 
 	if len(fieldSet) > 0 {
 		var b bytes.Buffer
-		b.WriteString("items(")
+		b.WriteString("prefixes,items(")
```

Comment:
Could you add a verification in the system tests for SetAttrSelection?

Reply:
Done. Really should convert the whole of TestIntegration_Objects to use table-driven style and subtests, but another time :)